EmmyLua/FrameXML/ChatFrame.lua

```diff
@@ -0,0 +1,155 @@
+---@type number
+NUM_CHAT_WINDOWS = 0
```

Comment:
Same here, why not `NUM_CHAT_WINDOWS = 10`? It's the same for Classic and Live iirc